tests/library/inspector/cli-codegen-1.spec.ts

```diff
@@ -624,6 +624,93 @@ await page.Locator(\"#age\").SelectOptionAsync(new[] { \"2\" });`);
     expect(message.text()).toBe('2');
   });
 
+  const clickMultipleSelectOption = async (locator: Locator, withCtrl = false) => {
+    const page = locator.page();
+
+    // Webkit can't click multiple select options
+    // https://github.com/microsoft/playwright/issues/32126
+    if (page.context().browser().browserType().name() === 'webkit') {
+      const elem = await locator.elementHandle();
```

Comment:
element handles are discouraged, there is a way to get bounding box from locator in one line.

Reply:
`boundingBox` returns `null` as it's assumed as not visible. That's why I use eval thing here